chingu-voyages/v37-bears-team-14
# Add member management project-setting page

# Changes

- Adds member management to project settings (under `/projects/{id}/settings/team`)
- Adds user selector modal
- Adds role selector for modifying team member roles

<img width="901" alt="Screen Shot 2022-02-24 at 5 48 45 PM" src="https://user-images.githubusercontent.com/8432061/155638585-f228c568-6cb0-4974-94d5-8d548ab53938.png">
<img width="901" alt="Screen Shot 2022-02-24 at 5 48 55 PM" src="https://user-images.githubusercontent.com/8432061/155638603-0fa31fd4-1db4-4c6b-bb16-fdb4c4be5436.png">


# Testing
- run app
- navigate to one of the projects that you own (or create a new one)
- navigate to the settings page for team
- try adding / removing members and check that it works

Closes #54
